(Psychologist now starts with a Lobotomy Kit) Redux

XRandomXManX opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

Thank 'Redux' for making a cameo after showing up on the Merriam-Webster Word-of-the-Day Podcast, January 1st.

## About The Pull Request

The Psychologist now starts with a little Lobotomy Kit.
It comes with new surgical items that are meant to be exclusive to them, but they can also be ordered from Cargo.
Since ALL Traumas caused by the Lobotomy are permanent, this nerfs any traumas that could completely take someone out of the game.
The Traumas nerfed are blindness, which is now nearsightedness, and Paralysis, which now cannot paralyze your whole body at once (Although you can still end up paralyzing both of your arms at once!).
This PR also removes the normal Lobotomy Procedure and has it replaced with the new one.

## Why It's Good For The Game

This will give the Psychologist some more use, as it lets them attempt to inflict special traumas onto their patients at the cost of potential mild or severe traumas.
I think this could be a really fun addition that would allow the Psychologist to both mess around and actually cure brain traumas.

## Changelog
:cl: 2Lucky
imageadd: Added Case, Orbitoclast, and Hammer Sprites
add: Added Lobotomy Case, Orbitoclast, Hammer, and Transorbital Lobotomy Procedure
del: Removed old Lobotomy
balance: Blindness Brain Trauma is now Nearsightedness.
balance: Paralysis Brain Trauma can no longer paralyze your entire body at once.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Commit:
moves all of the psychologists prescriptions to a box

Commit:
adds hammer and orbitoclast sprites

Commit:
adds in orbitoclast and hammer tools

Commit:
gives the hammer a significant buff

Commit:
adds in sprite for lobotomy tools case

Commit:
adds lobotomy kit, adds weight class for hammer

Commit:
fixes many small misprints, adds briefcase to psycologists bag

Commit:
adds in transorbital lobotomy procedure. Last touches as well.

Commit:
adds an empty line at the end of organic steps

Co-authored-by: uomo <51800976+uomo91@users.noreply.github.com>

Commit:
adds an empty line at the end of briefcase.dm

Co-authored-by: uomo <51800976+uomo91@users.noreply.github.com>

Commit:
adds a line at the end of transorbital_lobotomy.dm

Co-authored-by: uomo <51800976+uomo91@users.noreply.github.com>

Commit:
removes repeat lines of code

Commit:
Mild Traumas caused by failed Transorbital Lobotomies can no longer be cured by more Lobotomies

Commit:
Fixes a misprint

Fixes a misprint that caused the initial hammering step to be described as picking to the player

Commit:
Orbitoclast is now pointy

Commit:
Merge branch 'psych-upgrade' of https://github.com/XRandomXManX/tgstation-1 into psych-upgrade

Commit:
switches the cure types with mild and severe

because I guess I was in some sort of freak accident the first time

Commit:
timechange1

Co-authored-by: Rohesie <rohesie@gmail.com>

Commit:
Update code/modules/surgery/transorbital_lobotomy.dm

Co-authored-by: Rohesie <rohesie@gmail.com>

Commit:
Update code/modules/surgery/organic_steps.dm

Co-authored-by: Rohesie <rohesie@gmail.com>

Commit:
Update code/modules/surgery/transorbital_lobotomy.dm

Co-authored-by: Rohesie <rohesie@gmail.com>

Commit:
Update code/modules/surgery/transorbital_lobotomy.dm

Co-authored-by: Rohesie <rohesie@gmail.com>

Commit:
Update code/modules/surgery/transorbital_lobotomy.dm

Co-authored-by: Rohesie <rohesie@gmail.com>

Commit:
fix

Commit:
spaces intead of tabs moment

Commit:
brian moment

Commit:
fixes inconsistent things, tool needs >4 force for drive

Commit:
Update transorbital_lobotomy.dm

Commit:
Update lobotomy.dm

Commit:
removes lobotomy

Commit:
Merge branch 'psych-upgrade' of https://github.com/XRandomXManX/tgstation-1 into psych-upgrade

Commit:
Delete transorbital_lobotomy.dm

Commit:
Revert "Merge branch 'psych-upgrade' of https://github.com/XRandomXManX/tgstation-1 into psych-upgrade"

This reverts commit 68ae95817fd85df9606940fd228928cca258396e, reversing
changes made to 9487f6fc8467e92e807aef6a318d1bad1e992f6d.

Commit:
Merge branch 'psych-upgrade' of https://github.com/XRandomXManX/tgstation-1 into psych-upgrade

Commit:
Update transorbital_lobotomy.dm

Commit:
removes references to lobotomy

Commit:
Merge branch 'psych-upgrade' of https://github.com/XRandomXManX/tgstation-1 into psych-upgrade

Commit:
i forgot to remove a reference

Commit:
No item, only drill

It is only 25% because you're hammering with it, dummy

Commit:
you can now order lobotomy tools from cargo

You can't order the case because the case has the ability to KO the same as a briefcase, making it a fairly good weapon, even if it does have reduced damage.

Commit:
adds groundwork for lobotomizer skillchip

I just don't know how to make it work! I want it to cause any lobotomy to work with any object. Just the final step.

Commit:
Revert "adds groundwork for lobotomizer skillchip"

This reverts commit 3ce7967d2f2548f7a92f11c4c53d7c3009ffba83.

Commit:
Nerfs Blindness and Paralze Traumas, Renerfs Lobotomy

XRandomXManX commented:
I really like the 43 separate commits I made and then undid as I had a mental breakdown. Everything is good, I swear.

Jared-Fogle commented:
>  This PR also removes the normal Lobotomy Procedure to be replaced with the new one.

...keep it in until you actually have a replacement...? Lobotomies are not uncommon.

XRandomXManX commented:
There is a replacement already. The PR is for adding the replacement. I'll reword that.

Jared-Fogle commented on code/modules/cargo/packs.dm:
```diff
@@ -1222,6 +1222,19 @@
 					/obj/item/roller)
 	crate_name = "surgical supplies crate"
 
+/datum/supply_pack/medical/lobotomy
```
I'm not sure I'm a fan of locking lobotomies without the psychologist behind cargo. Are you sure you don't want to put the tools behind the fabricator? Are you sure it's necessary to remove the normal lobotomy in favor of this, anyway?

XRandomXManX replied:
I think it is uncommon enough for cargo to be a fine way of getting it without it being something every doctor can add to their belt.

Jared-Fogle commented on code/modules/cargo/packs.dm:
```diff
@@ -1222,6 +1222,19 @@
 
+/datum/supply_pack/medical/lobotomy
+	name = "Lobotomy Tools Crate"
+	desc = "Comes with an Health Scanner, Orbitoclast, Hammer, Cautery and some surgical drapes (as well as some instructions, and morphine)!"
```
```suggestion
	desc = "Comes with a health scanner, orbitoclast, hammer, cautery, and some surgical drapes (as well as some instructions and morphine)!"
```

Commit:
capital letters! panik!

Co-authored-by: Jared-Fogle <35135081+Jared-Fogle@users.noreply.github.com>

XRandomXManX commented:
I REMEMBERRRRRRRRRRRRRRRRRRRRR. FOR THE PAST WEEK I'VE BEEN WONDERING WHAT I FORGOT TO ADD, I THOUGHT IT WAS A SUICIDE FUNCTION BUT I WAS WRONG! PREPARE YOURSELVES!

Commit:
The Psychologist can now communicate with heavily brain damaged people

XRandomXManX commented:
Anyways, addressing your concerns, although a lot of brain damaging scenarios require a lobotomy, you do not encounter these brain damaging scenarios too often. Having the Lobotomy Kit locked behind Cargo is so the Psychologist cannot be undermined by a Lathe, but if Brain Damage suddenly becomes a problem on the Station, it can be dealt with.

ATH1909 commented on code/datums/brain_damage/severe.dm:
```diff
@@ -69,9 +69,6 @@
 		paralysis_type = pick("full","left","right","arms","legs","r_arm","l_arm","r_leg","l_leg")
```
```suggestion
		paralysis_type = pick("left","right","arms","legs","r_arm","l_arm","r_leg","l_leg")
```

ATH1909 replied:
don't remove the "full" location entirely, just remove it from the list of randomly selectable locations (and add a comment saying that you've done this). admins and such might want to be able to give people full paralysis brain traumas.

XRandomXManX replied:
Yikes. I should have caught that.

Commit:
Update code/datums/brain_damage/severe.dm

Co-authored-by: ATH1909 <42606352+ATH1909@users.noreply.github.com>

ATH1909 commented on code/modules/research/designs/medical_designs.dm:
```diff
@@ -665,13 +665,6 @@
 	research_icon_state = "surgery_any"
 	var/surgery
 
-/datum/design/surgery/lobotomy
-	name = "Lobotomy"
-	desc = "An invasive surgical procedure which guarantees removal of almost all brain traumas, but might cause another permanent trauma in return."
-	id = "surgery_lobotomy"
-	surgery = /datum/surgery/advanced/lobotomy
-	research_icon_state = "surgery_head"
```
removing normal lobotomies means that abductors can no longer lobotomize people without mugging the psychologist or cargo, and means that you can't speed up the chain lobotomization process as much with alien (or advanced) tools.

XRandomXManX replied:
Time to make an Alien Version of these single-case tools. I don't feel bad about this because of the Chaplain's 50 Null Rods.

ATH1909 commented on code/modules/surgery/organic_steps.dm:
```diff
@@ -138,3 +138,37 @@
+//drive through bone
+/datum/surgery_step/drive
+	name = "drive through bone"
+	implements = list(TOOL_HAMMER = 100, TOOL_DRILL = 25)
```
this should have at least one ghetto option that isn't a surgical tool

ATH1909 commented on code/game/objects/items/storage/briefcase.dm:
```diff
@@ -48,3 +48,24 @@
+
+/obj/item/paper/guides/lobotomy
+	name = "Lobotomies For Dummies"
+	info = "Doing a Lobotomy is easy.<br> Target the eyes, use your drapes, orbitoclast, hammer, orbitoclast, then finish off with a cautery.<br> However, to retrieve special powers from this is a little harder.<br> You must fail the lobotomizing step (second orbitoclast) to cause some traumas.<br> If these traumas are too bad, do the step correctly (Removes Severe Traumas by just cutting off more o ...
```
SUCCEEDING on a lobotomy will also apply a trauma.

also, correct me if I'm wrong, but I don't think that succeeding on a lobotomy step actually heals any brain traumas, which is probably important to note, especially seeing as how failing a lobotomy does 80 brain damage.

XRandomXManX replied:
Sorry, this is outdated information. I'll rewrite it.

Commit:
can now drive through bone with wrench

ATH1909 commented on code/modules/surgery/transorbital_lobotomy.dm:
```diff
@@ -0,0 +1,63 @@
+	return target.getorganslot(ORGAN_SLOT_BRAIN) ? TRUE : FALSE
+
+
+/datum/surgery_step/tlobotomize
```
why'd you name this path "tlobotomy" if there's no "lobotomy" path to compete with anymore?

ATH1909 commented on code/modules/surgery/organic_steps.dm:
```diff
@@ -138,3 +138,37 @@
 	return ..()
+
+//pick bone
+/datum/surgery_step/pick
```
why are these two steps outside of the transorbital lobotomy file? they're not used in any other surgeries.

ATH1909 commented on code/modules/surgery/transorbital_lobotomy.dm:
```diff
@@ -0,0 +1,63 @@
+/datum/surgery_step/tlobotomize
+	name = "perform transorbital lobotomy"
+	implements = list(TOOL_ORBITOCLAST = 75, /obj/item/melee/transforming/energy/sword = 50, TOOL_SCREWDRIVER = 15, /obj/item/stack/rods = 10)
+	time = 7.5 SECONDS
```
another downside to removing normal lobotomies: you can't speed up any of the steps (aside from the last one) of a transorbital lobotomy using alien and/or advanced tools, because there aren't any advanced or alien equivalents for orbitoclasts and hammers.

ATH1909 commented on code/modules/surgery/organic_steps.dm:
```diff
@@ -138,3 +138,37 @@
+//drive through bone
+/datum/surgery_step/drive
+	name = "drive through bone"
+	implements = list(TOOL_HAMMER = 100, TOOL_DRILL = 35, TOOL_WRENCH = 20)
```
I'd go for TOOL_MINING here instead of TOOL_WRENCH (improvised pickaxes exist, so this makes it still a feasible "ghetto option")

ATH1909 replied:
OR you can go for any item with a sharpness of "SHARP_POINTY", if you want prisoners and such to be able to do this

XRandomXManX replied:
Thanks!

Commit:
fixes

Commit:
allows for mining tools to be used instead of wrenches for driving through bones

Commit:
Update transorbital_lobotomy.dm

msgerbs commented:
Does this mean I can do back-alley lobotomies with powertools?

XDTM commented:
I have nothing against psychologists being able to perform easy lobotomies, but i can't see why would it be necessary to remove regular lobotomies as well as nerfing existing traumas. The bad rolls are there to provide more risk to reckless brain experimentation, after all.

Winterous commented:
Adding the aphasia language to psychologist is a genius change.

REMOVING regular lobotomy seems really unnecessary. It makes medborgs unable to safely perform lobotomies, and it doesn't make sense logically. The transorb one can be done without research and without an operating computer, that's the advantage it has.

Nerfing blindness and paralysis is completely unnecessary as well. First of all full body is like a 1/9 chance when you roll paralysis, so it's incredibly unlikely; and becoming crippled are just the risks of lobotomy, can you cope with what you have, or will you roll the dice on getting something worse? Neither are a round remove, since you can become a borg and regain what you lost; also there ARE methods of healing magic traumas, they're just very out of the way (need a miner to give you a legion core).

ATH1909 commented:
> Adding the aphasia language to psychologist is a genius change; I'd say it makes sense for them to have the druggie language as well, from years of experience counseling junkies.
> 
> REMOVING regular lobotomy seems really unnecessary. It makes medborgs unable to safely perform lobotomies, and it doesn't make sense logically. The transorb one can be done without research and without an operating computer, that's the advantage it has.
> 
> Nerfing blindness and paralysis is completely unnecessary as well. First of all full body is like a 1/9 chance when you roll paralysis, so it's incredibly unlikely; and becoming crippled are just the risks of lobotomy, can you cope with what you have, or will you roll the dice on getting something worse? Neither are a round remove, since you can become a borg and regain what you lost; also there ARE methods of healing magic traumas, they're just very out of the way (need a miner to give you a legion core).

I agree with this, except for the part about the trauma nerfs. Remember that you can also roll these traumas from the spontaneous brain trauma event.

XDTM commented:
I can agree with removing total paralysis, at least as a random trauma result; but traumatic blindness is not on the same level of uninteractability and should be kept as a 'bad luck' roll in my opinion, since it's one of the staple 'brain-damage-can-make-you-lose-basic-human-functions' seen in other media alongside loss of speech, language, and memory.
A trauma that causes nearsightedness should definitely not be in the severe trauma pool, in any case, and should be moved to mild.

Winterous commented:
Yeah I suppose full body paralysis is a little, too severe, to gain randomly. Removing "full" from the pick() is fine, but don't remove it from the switch(), might as well leave it in as an option for future use or admin fuckery.

XRandomXManX commented:
I really like making new things, but I don't think my brain can keep up with a lot of this. Defending my PR, adding new things, and removing things I already changed is a lot and I've been really busy lately. Thanks for the feedback but I don't think I have the mental capacity to make pull requests like this.
